### PR TITLE
Closes #237: deprecation

### LINKED ISSUE #237: Next stage of deprecation cycle for `derive_var_afeye()` arugments
### Background Information

`lat_vals` and `dataset_occ` argument now need to be totally removed as we are in [Phase 3 of the deprecation cycle](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html?#deprecation)

Note: This issue can be started after the 1.1 release in June 2024

### Definition of Done

- `lat_vals` and `dataset_occ` argument totally removed from `derive_var_afeye()` function code 
- run `devtools::document()` to update function documentation
- tests that check for deprecation of these arguments deleted
- NEWS.md updated to say that these arguments have reached the final stage of the deprecation cycle and have now been deleted

### PR DESCRIPTION
**Pull request by ukey1025**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update).
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files.
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions]
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately.
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiralophtha (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiralophtha site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralophtha/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`.
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`.
- [x] Link the issue so that it closes after successful merging.
- [x] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
-# admiralophtha 1.1.0.9004
+# admiralophtha 1.2.0.
+
+## Updates to Functions
+
+- The following function arguments of `derive_var_afeye()` are removed (#237):
```
Comment (manciniedoardo) [inline NEWS.md:5]:
```suggestion
- The following function arguments of `derive_var_afeye()` are removed following the end of their deprecation cycle.(#237):
```

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Thanks @ukey1025.

- Please update the roxygen header of `derive_var_afeye,R` to remove references to `dataset_occ` and `lat_vals`.
- Please remove the deprecation tests for these arguments as well (see [here](https://github.com/pharmaverse/admiralophtha/blob/main/tests/testthat/test-derive_var_afeye.R)) as they will fail otherwise.

**Comment by manciniedoardo** [comment]
@ukey1025:

- please fix the code style check by running: `styler::style_file("R/derive_var_afeye.R")`
- please fix the man pages check by running: `roxygen2::roxygenize('.', roclets = c('rd', 'collate', 'namespace'))`

**Comment by manciniedoardo** [comment]
@ukey1025 sorry it took me a while to get to this. I found what the issue is: you correctly removed `lat_vals` as an argument to `derive_var_afeye()` but line 67 of `derive_var_afeye()` should also be removed as the object doesn't exist anymore.

**Comment by manciniedoardo** [comment]
@ukey1025 I think the remaining error was due to the fact that there were still reference to `lat_vals` in the body of the function - i have gone ahead and removed that for you, which should fix the final issues.

**Review by manciniedoardo — APPROVED** [review]
LGTM @ukey1025 thanks!
